modules/oe_content_project/src/Commands/CopyBudgetValues.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_project\Commands;
+
+use Drupal\Core\Batch\BatchBuilder;
+use Drupal\Core\DependencyInjection\DependencySerializationTrait;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Copies the project budget values from the old field to the new field.
+ *
+ * This command can be optionally executed to copy over values from old field.
+ */
+class CopyBudgetValues extends DrushCommands {
+
+  use StringTranslationTrait;
+  use DependencySerializationTrait;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The messenger.
+   *
+   * @var \Drupal\Core\Messenger\MessengerInterface
+   */
+  protected $messenger;
+
+  /**
+   * The node storage.
+   *
+   * @var \Drupal\node\NodeStorageInterface
+   */
+  protected $nodestorage;
+
+  /**
+   * UpdatePersonData class constructor.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager, MessengerInterface $messenger) {
+    parent::__construct();
+
+    $this->entityTypeManager = $entityTypeManager;
+    $this->messenger = $messenger;
+    $this->nodeStorage = $this->entityTypeManager->getStorage('node');
+  }
+
+  /**
+   * Triggers the field value copy.
+   *
+   * @command project-budget-copy-values:run
```

Comment:
We need to namespace commands with our project name, like oe-content:project-budget-copy-values